spaceship/lib/spaceship/two_step_or_factor_client.rb

```diff
@@ -123,14 +123,25 @@ def handle_two_factor(response, depth = 0)
       code_length = security_code["length"]
 
       puts("")
-      puts("(Input `sms` to escape this prompt and select a trusted phone number to send the code as a text message)")
-      code_type = 'trusteddevice'
-      code = ask("Please enter the #{code_length} digit code:")
-      body = { "securityCode" => { "code" => code.to_s } }.to_json
-
-      if code == 'sms'
+      if ENV["SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER"]
```

Comment:
> Do we also want to make sure its not an empty string? 🤔

I chose _not_ to do that, because as soon as `SPACESHIP_2FA_SMS_DEFAULT_PHONE_NUMBER` is set the user wants this code to execute. The output should hopefully give them enough indication to discover that the variable is empty.

Reply:
@janpio Yeah yeah, might be good to say something if not empty! Empty string things are usually hard to debug otherwise 🙃

Reply:
Done.